src/Symfony/Bundle/FrameworkBundle/Resources/config/router_dev.php

```diff
@@ -21,6 +21,11 @@
  * @author: Albert Jessurum <ajessu@gmail.com>
  */
 
+// Workaround https://bugs.php.net/64566
+if (ini_get('auto_prepend_file') && !in_array(realpath(ini_get('auto_prepend_file')), get_included_files(), true)) {
+    require ini_get('auto_prepend_file');
+}
```

Comment:
To be a real and exact workaround, we must replicate the behavior of the native feature, thus doing this first